client/src/components/Messaging/Message/ChartMessage.tsx

```diff
@@ -0,0 +1,26 @@
+import { Typography } from '@mui/material';
+import { useStyles } from './useStyles';
+
+interface props {
+  sender: string;
+  message: string;
+}
+const ChatMessage = ({ sender, message }: props): JSX.Element => {
+  const classes = useStyles();
+  const messageClass = sender === 'me' ? classes.sent : classes.received;
+
+  const textStyle = messageClass.split('-')[1] === 'sent' ? classes.textSent : classes.textRecieved;
+
+  return (
+    <>
+      <div className={`message ${messageClass}`}>
+        <img src="https://cdn.pixabay.com/photo/2021/09/12/18/07/robin-6619184_960_720.jpg" />
```

Comment:
This element is missing an `alt` attribute.

Reply:
Could we address this comment?

Reply:
It was addressed sir